src/pages/maintenance/MaintenancePage.tsx

```diff
@@ -0,0 +1,25 @@
+import DTEHeader from "../../components/Shared/UI/DTETypography/DTEHeader/DTEHeader";
+import DTEContent from "../../components/Shared/UI/DTETypography/DTEContent/DTEContent";
+
+const MaintenancePage = () => {
+  return (
+    <div role="main" id="main">
+      <div className="inset-text">
+        <DTEHeader as="h1">Sorry, the service is unavailable</DTEHeader>
+
+        <DTEContent>
+          You will be able to use the service from 9am on Tuesday 16th January
```

Comment:
Ideally this would be a general purpose maintenance page without hard-coded date / time.
At a minimum this must be removed after this release.

If we need to use the maintenance page in an emergency we can't replace this text without another deployment which may not be possible or desirable.